cwm_worker_operator/domains_config.py

```diff
@@ -233,6 +233,7 @@ def __init__(self, data, domains_config, request_hostname=None, is_data_from_cac
             request_data['__request_hostname'] = request_hostname
             self._last_update = request_data["__last_update"] = common.now().strftime("%Y%m%dT%H%M%S")
             domains_config.keys.volume_config.set(request_worker_id, json.dumps(request_data))
+            domains_config.keys.volume_config.set(request_hostname, request_worker_id)
```

Comment:
it's a good idea to do it this way, but it should be in a new key, otherwise it may cause unexpected problems and complicates the logic. Also, it should use the instance id (`self.id` - because request_worker_id may not always be set.

e.g. `domains_config.keys.volume_config_hostname_worker_id.set(request_hostname, self.id)`

when you add a new key, be sure to also delete it and handle all the lifecycle of key, you should do a global search for the volume_config key and make sure all of the same operations are handled for the new key as well

another issue here is that the code depends on request_worker_id being set, but this key should be set regardless of that - under an new "if" statement

also, the condition `(request_hostname is not None and data.get('__request_hostname') != request_hostname)` - is not relevant for storing this key